collections/_posts/2023-02-27-ConvNextV2.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: review
+title: "ConvNeXt V2: Co-designing and Scaling ConvNets with Masked Autoencoders"
+tags: deep-learning CNN transformer segmentation classification object-detection attention
+author: "Pierre Rougé"
+cite:
+    authors: "Sanghyun Woo, Shoubhik Debnath, Ronghang Hu, Xinlei Chen, Zhuang Liu, In So Kweon, Saining Xie"
+    title:   "ConvNeXt V2: Co-designing and Scaling ConvNets with Masked Autoencoders"
+    venue: arXiv
+pdf: "https://arxiv.org/abs/2301.00808"
+---
+
+# Notes
+
+* **Important : See this [review](https://creatis-myriad.github.io./2023/02/23/Conv-Next.html) to understand ConvNeXt V1**
+* Also you can refer to this [review](https://creatis-myriad.github.io/2022/08/31/MAE.html) about Masked Autoencoders (MAE)
+
+# Highlights
+
+* ConvNeXt V1 was focused on supervised learning only, however, similarly to transformers, convolution based models can benefit from self-supervised learning techniques (such as MAE)
+* Simply apply self-supervised learning methods (i.e. MAE) to ConvNeXt leads to sub optimal results
+* **This article proposes a fully convolutional masked autoencoder framework (MAE) and modifies the ConvNeXt architecture with Global Response Normalization (GRN) layers**
+
+
+
+# Fully convolutional masked autoencoder
+
+![](/collections/images/convnextV2/framework.jpg) 
+
+*Figure 1 : Fully convolutional masked autoencoder design*
+
+* They use a random masking strategy with a masking ratio of 0.6 
+
+* They use ConvNeXt as the encoder
+
+* Unlike transformers you can't simply remove masked patches from the image as the 2D structure of the image must be preserved for convolution
+
+* Also naive solutions such as replacing masked patches by *masked tokens* don't perform well in practice
+
+* The idea here is to see the masked image as a sparse data. Based on that, is natural to use sparse convolution that will operate only on the visible pixels
+  
+  | *Note 1 : if the center pixel of the convolution is masked then the convolution will not operate and just return a masked pixel*
```

Comment:
The pipe (`|`) is not a supported syntax in Markdown, meaning it won't lead to reproducible formatting depending on the exact parser.

Therefore, I would suggest to use quote blocks instead, which are not exactly the same but that are [recommended for similar uses](https://stackoverflow.com/questions/25654845/how-can-i-create-a-text-box-for-a-note-in-markdown) (e.g. Olivier already uses them in his own posts).

```suggestion
> Note 1 : if the center pixel of the convolution is masked then the convolution will not operate and just return a masked pixel
```